modules/article/views/recent.html

```diff
@@ -1,7 +1,14 @@
 {% if data.articles.length %}
   {% for article in data.articles %}
-    <h4><a href="{{ article._url }}">{{ article.title }}</a></h4>
-    <article>{% area article, 'blurb' %}</article>
+    <div class="article-excerpt">
+      {% if not article._url %}
+          <p class="meta purple"><em>To link to this article make sure a Article Index page is published</em></p>
```

Comment:
Let's be more direct since we know that it's not there. "Make sure" leaves it less certain.

"You will need to publish an Article Index page for article links to work." or something.